src/Stringable.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Patchlevel\EventSourcing;
+
+use Patchlevel\EventSourcing\Serializer\Normalizer\StringableNormalizer;
+
+#[StringableNormalizer]
+interface Stringable
```

Comment:
Not sure if we should mark this also as experimental since this is only introduced because of dcb

Reply:
👍